docs/features/promotions.md

```diff
@@ -5,6 +5,23 @@ Currently you can apply a percentage or flat amount promotion for each product i
 1. Enable the _Orchard Core Commerce - Promotion_ feature.
```

Comment:
I have this error message right after setting up the site with the Orchard Core Commerce - Development recipe:
![image](https://user-images.githubusercontent.com/80963259/217139054-53558750-84f9-4ac6-b1d3-e9212b307214.png)